### PR TITLE
Improved docs regarding custom events usage

### PR DESCRIPTION
**Pull request by JSLegendDev**
In the Kaboom.js docs, it's not clear how to use the `on()` function used to define custom events.
I tried to improve the docs to make usage more clear.

### CODE REVIEW

**Comment by lajbel** [comment]
I would also mention that there's a list of avaible events, like ground for Body, we should expose it in types too!

**Review by lajbel — CHANGES_REQUESTED** [review]
You should add a mention here about that there's no only custom events, there's also default events, ground is one, and there's others. So maybe I would change it and also mention what are the defaults events.

It's a fact that we don't expose the events very well, maybe another PR for the repo

**Comment by JSLegendDev** [comment]
I opted to just add to the existing doc that you can have custom events with an example instead of removing what was there before.

File: src/types.ts [hunk 1]
```diff
@@ -668,6 +668,17 @@ export interface KaboomCtx {
 	 *     destroy(bomb)
 	 *     addKaboom(bomb.pos)
 	 * })
+	 * 
+	 * // a custom event can be defined manually
+	 * // by passing a name and a callback function
+	 * on("talk", (message, posX, posY) => {
```
Comment (lajbel) [inline src/types.ts:674]: indentation on every component here

File: src/types.ts [hunk 2]
```diff
@@ -668,6 +668,17 @@ export interface KaboomCtx {
...
+	 *     add([text(message), pos(posX, posY - 100)])
+	 * })
+	 * onKeyPress("Space", () => {
```
Comment (lajbel) [inline src/types.ts:677]: Space in lowercase

**Review by lajbel — CHANGES_REQUESTED** [review]
Only a few things, but looking good

**Comment by JSLegendDev** [comment]
@lajbel Addressed feedback and fixed spacing issues.

**Review by lajbel — APPROVED** [review]
looking good!

**Comment by lajbel** [comment]
Thank you for your contribution to documentation! 🔥

**Pull request merged**